haiku-apps/simplebackup/simplebackup-2.0.1.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Graphical interface to run the zip command"
+DESCRIPTION="This application makes it easy to create and update backups of \
+large amounts of data.  For example, you can create a backup of your home \
+directory and run this application regularly to make sure the backup is up to \
+date.  SimpleBackup also accepts command line options, so it is easily \
+scriptable and can be used with scheduling applications."
+HOMEPAGE="https://perelandra0x309.github.io/simplebackup"
+COPYRIGHT="2010-2016 Brian Hill"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/Perelandra0x309/simplebackup/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="463df905df35a74e81127d1c28d20a1f2c94ff1b89f6939abf081ecc70e60c62"
+SOURCE_DIR="simplebackup-${portVersion}"
```

Comment:
It's a good idea to have a gcc4hybrid and 64bit Haiku for testing/building.
The first thing I do however, is to build the gcc4-as-2nd-arch in the standard gcc2hybrid with "hp simplebackup_x86". If that fails badly, i.e. not easily fixable, I don't have to bother with testing under primary gcc4, let alone 64bit, and can justify a "!x86" in the ARCHITECTURES.